Makefile

```diff
@@ -17,7 +17,7 @@ SRC ?= github.com/kubernetes-sigs/cri-tools
 TAG ?= ${GITHUB_ACTION_TAG}
 
 ifeq ($(TAG),)
-TAG := v1.26.1$(BUILD_META)
+TAG := v1.28.0$(BUILD_META)
```

Comment:
Not that it matters for tag CI, but we should have the same default TAG value in both the Dockerfile and the Makefile.